README.md

```diff
@@ -41,11 +41,11 @@ This is the most basic case.
            Gross Salary: £416.67
 
 Calculation rules:
- * Monthly Gross Salary: The monthly gross salary is the employee's annual gross salary divided by 12
+ * "Gross Salary" on a salary slip is equal to the employee's annual gross salary divided by 12.
```

Comment:
I think this kata is interesting, but at the same time it has too many moving parts. In this part we mention about the calculation rules. From my perspective "divided by 12" is more the calculation rule. The language we speak should be introduces in the specification. I tried to change the descriptions to have more information, but they look clumsy (maybe when I find some time I will create another PR). 

I think it is good point to start discussion what we want to achieve in this kata.